### PR TITLE
make ADAL PR pipeline CyberEO compliant.

### PR DESCRIPTION
**Pull request by p3dr0rv**
Add lint 
Add CodeQL
separate workload into jobs
image

https://identitydivision.visualstudio.com/Engineering/_build/results?buildId=1128005&view=results

### CODE REVIEW

File: adal/build.gradle [hunk 1]
```diff
@@ -182,8 +182,8 @@ dependencies {
 
 task sourcesJar(type: Jar) {
     from android.sourceSets.main.java.srcDirs
-    classifier = 'sources'
-    destinationDir = reporting.file("$project.buildDir/outputs/jar/")
+    archiveClassifier  = 'sources'
+    getDestinationDirectory().set(reporting.file("$project.buildDir/outputs/jar/"))
```
Comment (fadidurah) [inline adal/build.gradle]: Is this related to lint work?

Comment (p3dr0rv) [reply adal/build.gradle]: code QL, it was deprecated so I change the syntax

File: adal/build.gradle [hunk 2]
```diff
@@ -182,8 +182,8 @@ dependencies {
 
 task sourcesJar(type: Jar) {
     from android.sourceSets.main.java.srcDirs
-    classifier = 'sources'
-    destinationDir = reporting.file("$project.buildDir/outputs/jar/")
+    classifier 'sources'
+    destinationDirectory = reporting.file("$project.buildDir/outputs/jar/")
```
Comment (p3dr0rv) [inline adal/build.gradle:186]: previous syntax was deprecated, so I changed it for CodeQl

**Review by fadidurah — APPROVED** [review]

**Review by melissaahn — APPROVED** [review]
